src/components/adjudications/AdjudicationsSection.jsx

```diff
@@ -37,7 +37,7 @@ class AdjudicationsSection extends React.Component {
     const { match: { params: { eventId, performanceId }}} = this.props;
     const { adjudications, loading } = this.state;
     const collectionName = `events/${eventId}/performances/${performanceId}/adjudications`;
-    const headings = ['Judge', 'Audio', 'Cumulative Score', 'Awards'];
+    const headings = ['Judge', 'Audio', 'Artistic Score', 'Technical Score', 'Cumulative Score', 'Awards'];
```

Comment:
Do we want to keep this consistent with variable names by using "Mark" instead "Score"?